src/test/java/com/rrules/RussianRouletteTest.java

```diff
@@ -0,0 +1,37 @@
+
+package com.rrules;
+
+import com.rrules.model.NewsType;
+import com.rrules.services.RussianRouletteService;
+import com.rrules.services.RussianRouletteServiceImpl;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import org.junit.jupiter.api.Test;
+
+public class RussianRouletteTest {
+    
+    RussianRouletteService rr = new RussianRouletteServiceImpl();
+    
+    @Test
+    public void contextLoads() throws Exception {
+	assertThat(rr).isNotNull();
+    }  
+    
+    @Test
+    public void shouldReturnCorrectTypeTest() {
+        assertThat(rr.randomSpin()).isExactlyInstanceOf(NewsType.class);
+    }
+    
+    @Test
+    public void shouldReturnOneInFiveProbabilityTest() {
+
```

Comment:
W tym tescie probujesz okielznac wyniki zwracane przez losowy generator, co przynajmniej czasem zakonczy sie bledem w tekscie, bo losowosc jest przypadkowa. Jest na to pewne obejscie, ktore mozemy razem sprobowac omowic na kolejnym spotkaniu, bo ciezko byloby je wytlumaczyc w komentarzu.